package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.6.1",
+  "version": "6.0.0",
```

Comment:
Maybe 5.7.0? Otherwise we'll have to upgrade another major for vue3?

Reply:
I'm considering it, but given the number of package changes, I'm thinking this will require a ton of regression testing... so I'm still thinking it's a major version update.

What do other devs think?

Reply:
These changes break backward compatibility, so I would say following the semantic versioning [practices](https://semver.org/) this is a major release

Reply:
Excellent point, Alex.

Reply:
I don't think it breaks backwards compatibility.  

If you were to revert the changes from this PR, 5.6.2 and "6.0.0" the application would still work using the same API.

This isn't a library like BCRS shared components or SBC Common components, so I think it should be treated differently.

Reply:
There are code changes in here (in the templates) that are not backward compatible.

That said, everyone now knows what I've done in here, so we don't _need_ to increase the major version number for visibility purposes. I'd be ok with going to 5.7.0 if the other reviewers agree.

Reply:
From a library perspective or an application perspective?

Reply:
I'll think about it and make a decision on Monday 😀